README.md

```diff
@@ -1,2 +1,4 @@
 # noodlemaps
 Better multi-stop path planning built on google maps
+
```

Comment:
hi bestie!

Reply:
> looks bad!

hello bestie! could you please elaborate? this isn't particularly useful feedback, likely stemming to the fact that you're a bad person

(copying from my comment)

Reply:
resolving bc im right and youre wrong B)